workflows/scripts/convert.py

```diff
@@ -1,22 +1,46 @@
 """
-script: covert.py
+script: convert.py
 
 converts sqlite (or other formats) into parquet files
 """
 import pathlib
-from typing import Union, List
+from typing import List, Union
 
 import cytotable
 
 
-def sqlite_to_parquet(
+def convert_to_parquet(
     input_file: Union[str, List[str]],
     out_path: str,
     target_ext: str,
     convert_configs: str,
 ):
+    """Takes in a file or a list of files that will be converted into parquet.
+
+
+
+    Parameters
+    ----------
+    input_file : Union[str, List[str]]
+        files or list of files converted into `.parquet` files
+    out_path : str
+        path where generated parquet files will be saved
+    target_ext : str
+        dictates which file format
+    convert_configs : dict
+        dictionary containing cytotable.convert() configs
+
+
```

Comment:
Minor comment: consider adjusting the newlines here (depending on what's aligned with linting).